ipaullly/sendIT
# [#161957214] Add email pattern validation

#### What does this PR do?
Adds a regex function in the `utilities` directory to check if the email in the input form follows accepted standard (i.e. name@mail.com). 
#### Description of Task to be completed?
- Define a regex function to ensure that the pattern of provided email is valid
- Add `if/else` conditional in registration view function to ensure emails are in the correct format, an error is thrown otherwise
#### How should this be manually tested?
clone repo into a local directory
`git clone --single-branch -b bg-email-pattern-validation-161957214 git@github.com:ipaullly/sendIT.git`
cd into the sendIT directory
activate virtual environment
`source venv/bin/activate`
install dependencies
`pip install requirements.txt`
power up Postman and send JSON with a wrongly formatted user email via a `POST register` endpoint to your server and you will see an error message that catches the wrong email pattern
